{{cookiecutter.project_slug}}/clients/web/vue3/.eslintrc.js

```diff
@@ -5,14 +5,18 @@ module.exports = {
   },
   extends: ['plugin:vue/vue3-essential', 'eslint:recommended', '@vue/prettier'],
   parserOptions: {
-    parser: 'babel-eslint',
+    parser: '@babel/eslint-parser',
+    requireConfigFile: false 
   },
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-unused-vars': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'vue/no-unused-components': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'vue/no-v-model-argument': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'vue/multi-word-component-names': ['error', {
+      'ignores': ['Home', 'Dashboard', 'Signup', 'Login']
```

Comment:
Kind of a hack to avoid more work; new upgraded linting rules requires components to have multiword names to avoid conflicting with built in html components. lmk if you want me to go through and actually rename everything

Reply:
I believe one of the main issues with `node 18` is the backend dev proxy was broken/failing 
https://stackoverflow.com/questions/74548318/how-to-resolve-error-error0308010cdigital-envelope-routinesunsupported-no
Is it working in this PR?

Reply:
seems to work fine to me:

![image](https://github.com/thinknimble/tn-spa-bootstrapper/assets/10493269/a169c619-bdd3-4d6b-96a5-55debc12909e)